Add docs on configuring SAML session lifetime (#2904)
pulumi/pulumi-hugo: themes/default/content/docs/pulumi-cloud/access-management/saml/okta.md

Once the IDP metadata descriptor has been saved, you are all set to log into Pulumi.

## Configuring Session Lifetime

> The Pulumi Cloud uses the `SessionNotOnOrAfter` attribute in the `AuthnStatement` element to configure the session lifetime. In order to configure this in Okta, you will need to use a [SAML assertion inline hook](https://developer.okta.com/docs/guides/saml-inline-hook/main/).

Reviewer: ```suggestion
The Pulumi Cloud uses the `SessionNotOnOrAfter` attribute in the `AuthnStatement` element to configure the session lifetime. To configure this in Okta, you must use a [SAML assertion inline hook](https://developer.okta.com/docs/guides/saml-inline-hook/main/).
```
